docs/root/start/sandboxes/skywalking_tracing.rst

```diff
@@ -0,0 +1,92 @@
+.. _install_sandboxes_skywalking_tracing:
+
+SkyWalking Tracing
+==================
+
+The SkyWalking tracing sandbox demonstrates Envoy's :ref:`request tracing <arch_overview_tracing>`
+capabilities using `SkyWalking <http://skywalking.apache.org>`_ as the tracing provider. This sandbox
+is very similar to the Zipkin sandbox. All containers will be deployed inside a virtual network
+called ``envoymesh``.
+
+All incoming requests are routed via the front Envoy, which is acting as a reverse proxy
+sitting on the edge of the ``envoymesh`` network. Port ``8000`` is exposed
+by docker compose (see :repo:`/examples/skywalking-tracing/docker-compose.yaml`). Notice that
+all Envoys are configured to collect request traces (e.g., http_connection_manager/config/tracing setup in
+:repo:`/examples/skywalking-tracing/front-envoy-skywalking.yaml`) and setup to propagate the spans generated
+by the SkyWalking tracer to a SkyWalking cluster (trace driver setup
+in :repo:`/examples/skywalking-tracing/front-envoy-skywalking.yaml`).
+
+When service1 accepts the request forwarded from front envoy, it will makes an API call to service2 before
+returning a response.
+
+Running the Sandbox
```

Comment:
you should remove this header (i removed the others while this PR was waiting to land)

see here https://storage.googleapis.com/envoy-pr/13060/docs/start/sandboxes/index.html